config.js

```diff
@@ -151,23 +116,35 @@ exports.updateFromArgv = function(argv) {
     // console messages
     // C - test exports warnings
 
-    testExportsWarnings();
+    warnIfNopauseIsMissing("ranked");
+    warnIfNopauseIsMissing("unranked");
```

Comment:
No need for general arg? Also, these can be called in a single function.

Reply:
Nevermind, it checks genral arg twice, in both calls. Mmm.

Reply:
should i also wrap this in a function?

Reply:
yeah that would be nice.

Reply:
roger 🎖️

Reply:
fixed